.github/workflows/test-check-tos.yml

```diff
@@ -0,0 +1,16 @@
+name: TEST - Gradle - Terms of Service approval verification
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+  actions: write
+  statuses: write
+
+jobs:
+  gradle-check-tos:
+    uses: ./.github/workflows/check-tos.yml
+    with:
+      tos-location: 'https://foo.bar/tos.html'
+      white-list: '*'
```

Comment:
If this is a required input, we should mark it as such the reusable action/workflow. If it's not required, we probably shouldn't be setting it in our test workflow.
(But I'm not convinced we need to expose this to users at all).

Reply:
It is an optional parameter, I'm using it in the test workflow to make it pass even if the contributor did not approve the TOS